audio_service/example/lib/main.dart

```diff
@@ -776,264 +779,192 @@ class MediaLibrary {
   static const albumsRootId = 'albums';
 
   final items = <String, List<MediaItem>>{
-    AudioService.browsableRootId: [
+    AudioService.browsableRootId: const [
       MediaItem(
         id: albumsRootId,
-        album: "",
-        title: "Albums",
+        album: '',
+        title: 'Albums',
         playable: false,
       ),
     ],
     albumsRootId: [
       MediaItem(
-        id: "https://s3.amazonaws.com/scifri-episodes/scifri20181123-episode.mp3",
+        id: 'https://s3.amazonaws.com/scifri-episodes/scifri20181123-episode.mp3',
         album: "Science Friday",
         title: "A Salute To Head-Scratching Science",
         artist: "Science Friday and WNYC Studios",
-        duration: Duration(milliseconds: 5739820),
+        duration: const Duration(milliseconds: 5739820),
         artUri: Uri.parse(
-            "https://media.wnyc.org/i/1400/1400/l/80/1/ScienceFriday_WNYCStudios_1400.jpg"),
+            'https://media.wnyc.org/i/1400/1400/l/80/1/ScienceFriday_WNYCStudios_1400.jpg'),
       ),
       MediaItem(
-        id: "https://s3.amazonaws.com/scifri-segments/scifri201711241.mp3",
+        id: 'https://s3.amazonaws.com/scifri-segments/scifri201711241.mp3',
         album: "Science Friday",
         title: "From Cat Rheology To Operatic Incompetence",
         artist: "Science Friday and WNYC Studios",
-        duration: Duration(milliseconds: 2856950),
+        duration: const Duration(milliseconds: 2856950),
         artUri: Uri.parse(
-            "https://media.wnyc.org/i/1400/1400/l/80/1/ScienceFriday_WNYCStudios_1400.jpg"),
+            'https://media.wnyc.org/i/1400/1400/l/80/1/ScienceFriday_WNYCStudios_1400.jpg'),
       ),
     ],
   };
 }
 
 /// This task defines logic for speaking a sequence of numbers using
 /// text-to-speech.
```

Comment:
I wanted the example to include at least one example of how to drive the audio playback logic with a loop, hence what you see. Originally the `Tts` wrapper was there to support a simple loop like this:

```dart
for (var n = 1; n <= 10; n++) {
  await _tts.speak("$n");
  await Future.delayed(Duration(milliseconds: 500));
}
```

Since then, flutter_tts has natively added support for awaiting the completion via the returned future if you pass in the appropriate option to request that behaviour, although now that's moot since I decided to change the example to make every utterance happen at a constant period. It's hard to read what exactly you've changed from the diff since you've both uncommented code and changed it, but I'm not sure if you've preserved this behaviour.
Things do get convoluted when handling next/previous controls in the loop, but that is

Reply:
your example was quite complicated (though completely reasonable because you created a while ago)

i drastically simplified it with new utils from `flutter_tts`, that is `setCompletionHandler` (see below)

i believe that is the loop behavior you wanted to demonstrate, however i'm not sure. you better run it and check out yourself if you are not sure

about constant time - we could remove it, but then seekbar wouldn't move at all - it's purely cosmetical

Reply:
The completion handler isn't new and I encapsulated it in the `Tts` wrapper since I wanted a more ergonomic API for waiting for completion - this has since been adopted by the flutter_tts author so there's no need for the `Tts` wrapper anymore. But at the same time I don't require that feature anymore since I intentionally changed the example to use a constant time on each utterance.

Using `setCompletionHandler` is not the same as a loop, what I wanted to demonstrate was an actual loop. E.g. a for or while loop.

Reply:
do we want to do something here? i would say i'm personally pleased with it at the moment